tests/test_resonance_scattering/test_resonance_scattering.py

```diff
@@ -1,60 +1,10 @@
 #!/usr/bin/env python
 
-import os
 import sys
-from subprocess import Popen, STDOUT, PIPE, call
-import filecmp
-import glob
-from optparse import OptionParser
+sys.path.insert(0, '..')
```

Comment:
Why does ``sys.path.insert(0, '..')`` need to be here and elsewhere? Looks like it is already in "testing_harness.py"